news/_posts/2020-12-09-pr-testing-on-prestashop.md

```diff
@@ -0,0 +1,179 @@
+
+### What happens next?
+
+Like with bug fixes, if the QA analyst finds any bug, they will comment the PR with all useful tips to help the author fix the bug.
+If the QA analyst notices an unspecified behavior (if the expected behavior is unclear, or for example, if it could be argued that an error should be displayed in a given scenario) they will ping the Product team and/or the maintainers to discuss this particular issue.
+
+If everything is fine, the QA analyst changes the label to “QA Approved” and the PR is merged by a maintainer, making it available on PrestaShop in the next build of the branch!
+
+Same as for bug fixes, these tests are tracked on an internal tool. The QA analyst might also update some of the test campaigns to add or modify test cases related to this improvement.
+
+# IV. Migration Pull Requests
+
+Migration Pull Requests are quite different to other ones: the purpose of these PRs is to move a page from the Legacy framework to the Symfony framework, while keeping the same features (and optionally adding some minor improvements).
+
+This makes the QA's work on these Pull Requests quite longer and harder compared to other PRs.
+
+### Requirements
+
+The main difference here is that the issue linked to the PR might be just an Epic listing all the items that should be migrated.
+
+For the “How to Test”, it’s sometimes only “Everything should work as in Legacy”. If there is a specification linked to this page, it should be added to the “How to Test”, to help the QA analyst make sure to check everything. If the migrated page is “hidden” (i.e. not accessible through the menu), the “How to Test” must also contain the URL to access it.
+
+### Test of a Migration Pull Request
+
+The test of a Migration PR is quite long: The QA analyst must compare the Legacy page and Migrated page, and verify that there are no errors and nothing is lost (information, links, buttons, etc.), using every configuration they can think of (multi-currencies, multi-languages, multi-store, using different parameters...).
+
+If there is a specification, the QA analyst will follow every point and check that it’s working fine (an example of a migration specification : ["Product listing" BO page](https://github.com/PrestaShop/prestashop-specs/blob/master/back-office/catalog/products/catalog-products-listing.md)). If there is not, they must find every detail by themselves.
+
+During this run the QA analyst might use the test campaign created for build releases, and also update these same tests if necessary.
+That’s why a test on a Migration PR takes longer than a “classic” one (sometimes several days).
+
+# Conclusion
+
+Quality Assurance work on Pull Requests can be really time-consuming, but it is critical to ensure that quality of PrestaShop software is ever increasing.
+Make sure that the Pull Requests and issues you submit are as thoroughly described as possible – it will help processing them quickly and seamlessly!
```

Comment:
```suggestion
Make sure that the Pull Requests and issues you submit are thoroughly described, it will help to process them quickly and seamlessly!
```